templates/mail/contact.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+{% block body %}
+  <h2>Submit a request</h2>
+  <span>Fields marked with an asterisk (*) are required.</span>
+
+  <div id="contactSupportForm"></div>
+{% endblock %}
```

Comment:
Thanks for dealing with my wacky vue integration. It's an experiment so we can discuss how we like this workflow later.